pkg/model/paths.go

```diff
@@ -29,24 +34,24 @@ type ArchivePathComponents struct {
 	ArchiveFileName string
 	LabelName       string
 	Context         string
+	DiamondID       string
+	SplitID         string
 }
 
-// GetArchivePathComponents yields all components from an archive path.
-//
-// NOTE: this function's design is converging on being able to return something meaningful
-// given any path in the metadata archive, not just those corresponding to bundles.
-//
-// The return value might be changed to an interface type in later iterations.
```

Comment:
with the last iteration of this comment being elided from the source and [yet another](https://github.com/oneconcern/datamon/pull/403#discussion_r379583010) instance of "let's use `interfaces` to represent sum/union types!" enthusiasm appearing in my commentary elsewhere, i'd like to have the vestiges of this link trail (into github comments, into git history) referenced from the issue tracker.

so i am doing just that!  creating a spin-off issue here and resolving this comment immediately.

Reply:
My point was that:
>  this function's design is converging on being able to return something meaningful
has no more justification, since this function _has converged_ on being able to return something meaningful.